compiler/cpp/src/thrift/generate/t_go_generator.cc

```diff
@@ -3463,6 +3550,112 @@ void t_go_generator::generate_serialize_list_element(ostream& out, t_list* tlist
   generate_serialize_field(out, &efield, prefix);
 }
 
+/**
+ * Compares any type
+ */
+void t_go_generator::generate_go_equals(std::ostream& out,
+                                           t_type* ori_type,
```

Comment:
nit: this indentation looks weird. usually you either align with `(`, or use fixed 2/4/6 spaces. this is neither.

I don't believe clang-format is used to generate this indentation either, at least it's not enforced by travis. when I run `clang-format -i --style=file compiler/cpp/src/thrift/generate/t_go_generator.cc` it made a ton of changes in this file.

Reply:
not sure is the `std::` making indentation looks weird. Indentations looks proper after I remove `std::` and apply `clang-format`.
To avoid unnecessary change, I only apply the output of clang-format on my changes in `compiler/cpp/src/thrift/generate/t_go_generator.cc`.